Added support for Vivo.sx

pkos98 opened the pull request:
Added support for Vivo.sx.
Btw, why does the manifest file contain the key `applications`?  
It is only mandatory for Firefox < 48.

maciex committed:
Use new plugin.video.youtube API

Fixes #11

maciex committed:
Allow XHR access to '*://www.googleapis.com/*'

Fixes #12

maciex committed:
New version 1.8.1

arturasokol committed:
Add support for pornhub.com - because why the hell not

predakanga committed:
Update Facebook video regex

At least in .au, there is no longer a " after ratelimit (see https://www.facebook.com/abcqanda/videos/10154336391111831/). Updated regex will match both forms.

Commit:
Change p2p-streams to more recent Plexus plugin to handle sopcast and acestream links

Commit:
Added support for Vivo.sx

Added support for Vivo.sx

Commit:
Removed unnecessary comments

Commit:
Added Vivo.sx to list of supported sites

kmichal commented:
Hi,

Thank you for the PR but you should probably do a PR to the author's repo (khloke/play-to-xbmc-chrome).
This is a fork I used to fix a bug.

Maciej-Sitarz-IBM committed:
Merge remote-tracking branch 'upstream/master'

maciex committed:
Fix SyntaxError: test for equality (==) mistyped

JavaScript strict warning: moz-extension://696dbf77-029f-4fb8-a1ad-29c376e78175/js/modules.js, line 585: SyntaxError: test for equality (==) mistyped as assignment (=)?
JavaScript strict warning: moz-extension://696dbf77-029f-4fb8-a1ad-29c376e78175/js/modules.js, line 587: SyntaxError: test for equality (==) mistyped as assignment (=)?

maciex committed:
Version bump

maciex committed:
Porting changes after upstream pull

Changes:
- Changed all 'chrome.extension.' occurances to 'chrome.runtime.'
- Rewritten all 'chrome.tabs.getSelected()' usages to 'chrome.tabs.query()'

Commit:
[#121] Workaround solution for event.target[matches] is not a function error

khloke committed:
Merge pull request #123 from JohnyGemityg/master

[#121] Workaround solution for event.target[matches] is not a func...

khloke committed:
Merge pull request #116 from predakanga/patch-1

Update Facebook video regex

khloke committed:
Merge pull request #100 from maciex/master

Fix YouTube playlist queueing in Firefox

khloke committed:
Merge pull request #114 from arturasokol/modules/pornhub

Add support for pornhub.com - because why the hell not

khloke committed:
Merge pull request #118 from jlopezinc/p2p-streams-to-plexus

Change p2p-streams to more recent Plexus plugin to handle sopcast and…

khloke committed:
Merge branch 'master' into vivo_sx
